lib/actions/actions.js

```diff
@@ -17,9 +18,11 @@ class Actions {
     assert(typeof fullMessageText === 'string');
     const commandMap = {
       add: new AddLinkToProjectAction(),
+      overview: new ProjectOverviewAction(),
     };
     if (fullMessageText === '') {
-      throw new TypeError('No Term Found');
+      const actionName = 'overview';
```

Comment:
I'm sorry I didn't remember this before, but I thought we decided `Actions.parseActionTerm` was responsible for providing the action name. One path through the code is more comprehensible than having this if condition–it will be more readable. Honestly, I don't think you need to check for `fullMessageText === ''` at all in this method.